cypress/e2e/manager_trip.cy.js

```diff
@@ -0,0 +1,26 @@
+import testUser from "../../credentials/managerCredentials.json"
+import BookingPage from "../pages/BookingPage.js"
+import exampleTour from "../support/tourNames.json"
+import LoginPage from "../pages/LoginPage.js"
+
+const login = () => {
+    cy.visit('/Account/Login?ReturnUrl=%2F')
+    LoginPage.submitLogin(testUser.login,testUser.pass)
+
+}
+
+describe('tests for adding tour function', () => {
+    it('Positive scenario - manager add polish tour', () =>{
```

Comment:
brak spacji po =>
zainstaluje sobie do visual studio eslint - on automaytcznie formatuje Ci kod, jest to standardowe narzedzie uzywane w developerce, czasem sa checki przy prkach gdzie nie mozesz wrzucic czegos do repo bo linter widzie gdzies bledy w formatowaniu kodu